### ISSUE TITLE
Investigate field-level help

### ISSUE THREAD

**Issue by mjherzog** [nexB/aboutcode-manager#273]
It would be very helpful to offer field-level help for the UI buttons and for column headers - either by mouse over or some icon. This is much more convenient for basic definition of functions and data as compared to linking to external wiki or other pages. I hope that there is something already available in the Electron domain - not suggesting that we write this feature on our own.

**Comment by Saichethan** [comment]
Is this issue still open. Can I work on it?

**Comment by majurg** [comment]
@Saichethan Yes this issue is available.

**Comment by TG1999** [comment]
@majurg can you explain this issue. How can I solve it ?

**Comment by chinmaym07** [comment]
Hey @majurg  Can i work on this issue. 
what info to be included in tooltips.?

**Comment by gat786** [comment]
I am just guessing that we can add tooltips to those elements and it would all work out fine?
Is there an exhaustive list of on which elements do we want this tooltip to show? 
Can I startup with a particular page and then start working there and then go one by one to other pages to add this tooltip everywhere?
Also if I start on say the table page I will have to add tooltips to headers of the table?
What will those tooltips be. 
Sorry if this is a long list of questions but I will need some help to move forward.

**Comment by steven-esser** [comment]
@gat786 Yes tooltips for those elements would be a great start. Unfortunately we do not have an exhaustive list. I think your approach of incremental addition would be a great start. I think the headers on the Table View would be great candidates for these tooltips; In this case, it would be a very short description of the corresponding header name. 

For instance, the tooltip for the `sha1` column header would be: "The sha1 checksum of a particular file." 

Perhaps this could be even shorter. We want to convert enough information in as little characters as possible IMO.

**Comment by gat786** [comment]
@majurg I just reviewed the codebase for a while. And the table view is created with the help of a `DataTables` library. I have found where the data is coming from but was not quite sure on how the tooltip will be appended to the `th` which is created by the library for the data. It will be very helpful if you can send a link guiding for the same.

**Comment by steven-esser** [comment]
@gat786 When running locally, you may be able to use the web view tools to find the element class name.
